docs/api/type/list32.rst

```diff
@@ -0,0 +1,13 @@
+
+.. xmethod:: datatable.Type.list32
+    :src: src/core/types/py_type.cc PyType::list32
+    :cvar: doc_Type_list32
+    :signature: list32(T)
+
+    List type with 32-bit offsets. In a column of this type, every element
+    is a list of types `T`.
```

Comment:
I would clarify what offsets mean in this context.

Reply:
Oh sure, we'd need to expand the documentation more when there's a better understanding of how this type works.